Creating a PokeCard component

erfemega opened the pull request.

erfemega committed:
initial commit

wdonet commented on src/App.js:
```diff
@@ -1,10 +1,16 @@
     <div className="App">
       <h1>Welcome to your second mini challenge!</h1>
+      <div className="pokemonsContainer">
+        <PokeCard pokemonData={mockPokemonData} />
```
Since there are not much elements inside the mocked data, you could send them as properties for having the code more descriptive itself
```suggestion
     const { name, sprites, video } = mockPokemonData;
      <PokeCard sprites={sprites} name={name} video={video} />
```

wdonet commented on src/components/PokeCard.js:
```diff
@@ -0,0 +1,18 @@
+        <img src={props.pokemonData.sprites.front_shiny} alt="Shiny" />
+      </div>
+      <div className="pokemonControls">
+        <a href={props.pokemonData.video} target="blank">
```
Following what I said before, it makes the code cleaner here using less navigation through the pokemon obj.
```suggestion
       PokeCard({ name, sprites, video })
        <a href={video} target="_blank" rel="noreferrer">
```
I know it is not much visible with this case but you'll see it for sure in more complex components.

wdonet commented on src/styles.css:
```diff
@@ -2,3 +2,44 @@
+  width: 100%;
+}
+
+.pokemonImages img {
```
nice 👌🏽  that you took care of the styling

erfemega committed:
Applying recommended changes
Destructuring mockPokemonData and pass properties separated to the PokeCard component
